src/tools/interop/idt/README.md

```diff
@@ -211,28 +187,18 @@ RPi users, as needed:
 ```
 idt capture -h
 
-usage: idt capture [-h] [--platform {Android}]
-                   [--ecosystem {PlayServices,PlayServicesUser,ALL}]
-                   [--pcap {t,f}]
-                   [--interface {wlp0s20f3,docker0,lo}]
-                   [--additional {t,f}]
+usage: idt capture [-h] [--platform {Android}] [--ecosystem {PlayServicesUser,PlayServices,ALL}] [--pcap {t,f}] [--interface {wlp0s20f3,lo,docker0,any}]
```

Comment:
Is that list of interfaces hardcoded? Or does it reflect the current set from the term where it's running? Is there any guidance on how to set that?

Reply:
The list of available platforms and ecosystems are dynamically loaded based on the contents of the ecosystem and platform packages. In the README, I've pictured the ecosystems and platforms currently committed.

Reply:
Added more info in the REAMDE here!